src/theme.js

```diff
@@ -90,7 +94,9 @@ export const colors = {
   ...bronzes,
   background: { ...backgroundColors },
   brand: { ...brandColors },
+  blurryShadowColor,
   border: { ...borderColors },
+  borderShadowColor,
```

Comment:
Do we need to include this and `blurryShadowColor` in the theme? It seems like it’s only used internally for the shadow elevations.

Reply:
I guess you are right. I was trying to expose the used colors through the color story but this is just used within the boxShadow elevations.